sw/device/tests/uart_baud_rate_test.c

```diff
@@ -24,11 +24,17 @@ static const uint32_t kBaseAddrs[4] = {
     TOP_EARLGREY_UART2_BASE_ADDR,
     TOP_EARLGREY_UART3_BASE_ADDR,
 };
-static const uint32_t kBauds[9] = {4800,   9600,   19200,  38400, 57600,
-                                   115200, 230400, 128000, 256000};
+static const uint32_t kBauds[11] = {4800,   9600,    19200,  38400,
+                                    57600,  115200,  230400, 128000,
+                                    256000, 1000000, 1500000};
```

Comment:
Perhaps worth demarcating with a comment those baud rates which are available only on Si from those that may be used everywhere, to be a bit more robust against the insertion or appending of additional rates in future?